tesseract/tesseract_planning/tesseract_process_managers/src/process_generators/iterative_spline_parameterization_process_generator.cpp

```diff
@@ -117,23 +101,18 @@ int IterativeSplineParameterizationProcessGenerator::conditionalProcess(ProcessI
   }
 
   Eigen::VectorXd velocity_scaling_factors = Eigen::VectorXd::Ones(static_cast<Eigen::Index>(flattened.size())) *
-                                             cur_plan_profile->max_velocity_scaling_factor;
+                                             cur_composite_profile->max_velocity_scaling_factor;
```

Comment:
Was thing moved from plan profile to composite profile?

Reply:
We were already pulling it from composite_profiles. The local variable we use here was just misnamed.

Reply:
Ah.